### PR TITLE
Notification when an imessage alias is removed

### PR DESCRIPTION
**Pull request by SeanReg**
#2581

Displays a notification with the alias(es) that were removed from an account

### CODE REVIEW

**Review by tneotia — CHANGES_REQUESTED** [review]

File: lib/services/backend/notifications/notifications_service.dart [hunk 1]
```diff
@@ -651,6 +651,44 @@ class NotificationsService extends GetxService {
...
+            android: AndroidNotificationDetails(
+              ERROR_CHANNEL,
+              'Errors',
+              channelDescription: 'Displays message send failures, connection failures, and more',
+              priority: Priority.max,
+              importance: Importance.max,
+              color: HexColor("4990de"),
+              ongoing: false,
+              onlyAlertOnce: true,
+              styleInformation: const BigTextStyleInformation('')
```
Comment (tneotia) [inline lib/services/backend/notifications/notifications_service.dart:685]: What is the purpose of this line?

Comment (SeanReg) [reply lib/services/backend/notifications/notifications_service.dart:683]: Allows the content text to wrap multiple lines

File: lib/services/backend/java_dart_interop/method_channel_service.dart [hunk 2]
```diff
@@ -211,6 +211,15 @@ class MethodChannelService extends GetxService {
         Logger.info("Answering FaceTime call");
         await intents.answerFaceTime(call.arguments["callUuid"]);
         return true;
+      case "imessage-alias-removed":
+        Map<String, dynamic> data = jsonDecode(call.arguments);
+        final addrLst = ((data['address'] ?? []) as List).map((a) => a as String).toList();
```
[the remark this answers is not in the document]
Comment (SeanReg) [reply lib/services/backend/java_dart_interop/method_channel_service.dart]: Updated to the new mechanism

**Review by tneotia — APPROVED** [review]
LGTM

**Pull request merged**